pipenv/utils.py

```diff
@@ -260,7 +261,8 @@ def convert_deps_to_pip(deps, r=True):
 
         # Support for files.
         if 'file' in deps[dep]:
-            dep = deps[dep]['file']
+            dep = '-e {0}'.format(deps[dep]['file'])
```

Comment:
i think the lockfile format should be file: too?

Reply:
i think this breaks support for files

Reply:
we don't want to magically make non-editable things editable

Reply:
2 things:

1. If we want to use *file:* uris in the lockfile should we be resolving the relative paths or can we leave them as relative?  
    * If we need to resolve them to use the URIs I would say we should probably just exclude them from the pipfile or change the spec to accommodate them somehow? I don't have a suggestion for this 
2. I agree this will probably break support for files by making non-editable things editable but I think I have a solution.  Let me push a change and see if you think it will work

Reply:
Ok the latest commit still doesn't use `file://` URIs but will handle editable files correctly and in a non-breaking manner (i.e. similarly to how we do it with VCS currently). The first pass was rushed.  I had to go out to dinner!

I'll need some input / would love some feedback about what to put in the lockfile.  If you're feeling brave we can revisit this tomorrow.  We just need to make a ruling on what to use as a name if we are moving between relative and absolute paths and what, if anything, we can hash.